akka-stream/src/main/scala/akka/stream/impl/FlexiRouteImpl.scala

```diff
@@ -31,6 +31,16 @@ private[akka] class FlexiRouteImpl[T, S <: Shape](_settings: ActorFlowMaterializ
   // needed to ensure that at most one element is emitted from onInput
   private val emitted = Array.ofDim[Boolean](outputCount)
 
+  override def preStart(): Unit = {
+    super.preStart()
```

Comment:
was tempted to wrap in try/finally yet currently we know it's the Actor#preStart so not needed I guess